src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -102,39 +104,44 @@ def __call__(self, inputs: Union[str, ImageSource, List[ImageSource]], batch_siz
 
         :param inputs:      inputs to the model, which can be any of the above-mentioned types.
         :param batch_size:  Maximum number of images to process at the same time.
+        :param show_progress_bar:   If True, show a progress bar.
         :return:            Results of the prediction.
         """
 
         if includes_video_extension(inputs):
-            return self.predict_video(inputs, batch_size)
+            return self.predict_video(inputs, batch_size, show_progress_bar)
```

Comment:
Please use explicit arguments here `show_progress_bar=show_progress_bar` and elsewhere